### PR TITLE
Update README_zh_hans.md

### PR DESCRIPTION
**Pull request by ZT-Pepite**
add sales information and emojis, correct some languages

### CODE REVIEW

**Comment by JasonMo1** [comment]
Thank you for your contribute!
I think we can add these emojis to English readme at the same time. 
And then, as you said on QQ, we can add YouTube and bilibili follower counts beside Apache LICENSE.

**Comment by JasonMo1** [comment]
Hmm, it seems there's some problem on formatting Markdown, 
I leave some comments about it.

File: docs/README_zh_hans.md [hunk 1]
```diff
@@ -23,33 +23,36 @@
     <a href="../README.md">English</a> | 简体中文
 </p>
 
-本项目旨在创建Zeal 8位计算机的软件模拟器（Zeal 8位计算机是一台自制的8位计算机，基于Z80 CPU）。
+**👏中文介绍———Zeal 8位计算机软件模拟器创建和使用指南👏**
+- **[点击此处访问在线模拟器](https://zeal8bit.github.io/Zeal-WebEmulator/)**
+- [Zeal 8位计算机]([url](https://zeal8bit.com/))是一台自制的8位计算机，基于Z80 CPU。
```
Comment (JasonMo1) [inline docs/README_zh_hans.md:28]: The Markdown format here is wrong

```suggestion
- [Zeal 8位计算机](https://zeal8bit.com/)是一台自制的8位计算机，基于Z80 CPU。
```

File: docs/README_zh_hans.md [hunk 2]
```diff
@@ -23,33 +23,36 @@
...
+- Zeal 8位计算机的[B站频道]([url](https://space.bilibili.com/1042658991))
```
Comment (JasonMo1) [inline docs/README_zh_hans.md:29]: Same here.

```suggestion
- Zeal 8位计算机的[B站频道](https://space.bilibili.com/1042658991)
```

File: docs/README_zh_hans.md [hunk 3]
```diff
@@ -23,33 +23,36 @@
...
+- [购买 Zeal 8位计算机]([url](https://www.tindie.com/products/zeal8bit/zeal-8-bit-computer-developer-edition-z80-based/))
```
Comment (JasonMo1) [inline docs/README_zh_hans.md:30]: And here

```suggestion
- [购买 Zeal 8位计算机](https://www.tindie.com/products/zeal8bit/zeal-8-bit-computer-developer-edition-z80-based/)
```

File: docs/README_zh_hans.md [hunk 4]
```diff
@@ -188,15 +191,15 @@ Video chip: implement 320x240 text-mode
...
+## 📝许可证
 
 `Z80.js` 在MIT licence下取得许可 (源自Molly Howell的存储库：[Z80.js](https://github.com/DrGoldfire/Z80.js))
 
 所有其他文件都在 Apache 2.0 许可证下分发。有关详细信息，请参阅“许可证”文件。
 
 您可以自由地将其用于个人和商业用途，但不得删除每个文件中存在的样板。
 
-# 联系方式
+## 📪联系方式
```
Comment (JasonMo1) [inline docs/README_zh_hans.md:202]: Thanks! I didn't found this before😂

File: docs/README_zh_hans.md (hunk 1 above)
Comment (ZT-Pepite) [reply docs/README_zh_hans.md:28]: Yes, thx for the modification

**Comment by ZT-Pepite** [comment]
thx for the modif. As far as adding Bilibli and YT followers counts, I will think about it

**Pull request closed without merging**